microservices-orders/docs/index.rst

```diff
@@ -0,0 +1,578 @@
+.. _microservices-orders:
+
+.. toctree:: 
+    :maxdepth: 2
+
+Developing Streaming Applications
+=================================
+
+This self-paced tutorial provides exercises for developers to apply the basic principles of streaming applications.
+
+========
+Overview
+========
+
+This is a small microservice ecosystem built with Kafka Streams. There is a related `blog post <https://www.confluent.io/blog/building-a-microservices-ecosystem-with-kafka-streams-and-ksql/>`__  that outlines the approach used.
+
+.. figure:: images/microservices-demo.jpg
+    :alt: image
+
+Note: this is demo code, not a production system and certain elements are left for further work.
+
+Microservices
+~~~~~~~~~~~~~
+
+The example centers around an Orders Service which provides a REST interface to POST and GET Orders.
+Posting an Order creates an event in Kafka.
```

Comment:
nit: we could mention the topic name is `Orders`

Reply:
Ack